AutoCollection/ClientRequestParser.ts

```diff
@@ -36,6 +39,8 @@ class ClientRequestParser extends RequestParser {
      */
     public onResponse(response: http.ClientResponse, properties?: { [key: string]: string }) {
         this._setStatus(response.statusCode, undefined, properties);
+        this.targetIKeyHash =
+            response.headers && response.headers[RequestResponseHeaders.targetInstrumentationKeyHeader];
```

Comment:
Will the result of the expression with '&&' operator be string? Sorry not very fluent in node, so just doing due diligence.